### DISCUSSION TITLE
Admin not allowed to create groups under real

### DISCUSSION THREAD

**Discussion by spgajjar** [keycloak/keycloak#30418] [Q&A]
Under my organization I have 7 realm and under each realms i have a few groups which has role mappings. I am able to add remove groups for all but one realm. For that realm when I go to groups section there is no add button at all. 

I checked realm setting for all and its identical as other realms where I am able to add the groups.

Am I missing some config somewhere? 

Can someone point me to right place?

**Comment by BastianPEIRE** [comment]
I have the exact same problem can someone answer ?
